### DISCUSSION TITLE
Folding something folds the entire file below

### DISCUSSION THREAD

**Discussion by noctopixi** [nvim-neorg/neorg#1389] [Q&A]
Hello. Sometimes, when I fold anything, such as a level 3 headers, it folds everything below it, including level 2 headers.

I'm not exactly sure how to reproduce it, but it only seems to occur when I have manually typed everything - if I copy and paste the very same text, folding works as expected (only the header that I have my cursor in is folded). It also seems to randomly fix itself if I modify the text in one of the headers.

I'm on Neorg 8.4.0. My OS is Fedora 39, and I'm using nvim 0.9.5. I installed Neorg via Lazy and use vhyrro's luarocks.nvim.

#### EXAMPLE:
Folding a level 3 header results in EVERYTHING below getting folded, including the level 2 header.

**Before**
![image](https://github.com/nvim-neorg/neorg/assets/164736486/8372d5db-2ff4-4c8d-838a-8cd0080e47ea)
**After**
![image](https://github.com/nvim-neorg/neorg/assets/164736486/3e93d90f-be20-4251-b5d8-20299344043c)

However, if I paste the very same set of headers and content, and try folding the same header, it only folds itself and its contents, not anything below it.

**Before**
![image](https://github.com/nvim-neorg/neorg/assets/164736486/5a04062e-8fcd-4bee-9c4b-937941b071fa)
**After**
![image](https://github.com/nvim-neorg/neorg/assets/164736486/dcf12359-fd39-4f36-a05e-3d58d2dd9cdf)

The only solution I have for my problem right now is deleting the entire file and pasting its contents again. Other than that, I'm not sure how to troubleshoot this. I'd appreciate any help - this is making folds difficult to work with. Thanks!

**Comment by lars-vc** [comment]
I have this issue also :)

**Comment by noctopixi** [comment]
@lars-vc Sorry to hear you're having it as well! The Discussion forum isn't very active, so I have opened a proper bug. You can follow it here: https://github.com/nvim-neorg/neorg/issues/1396

I'm leaving this thread open in case someone else has had this issue and has successfully fixed it.
